frontend/packages/topology/integration-tests/features/topology/topology-editing-app-node.feature

```diff
@@ -50,3 +50,155 @@ Feature: Editing an application
+              And user will see "Deploy image from an image stream tag" checkbox under Images with project, image stream and tag section associated with it
+              And user will see Image name section under Images with checkbox "Deploy image from an image stream tag" unchecked
+              And user will see "Show advanced image options" and "Show advanced container options" in case of image stream option
+              And user will see advanced option for "Pause rollouts" and "Scaling"
+
+
+        @regression @manual
+        Scenario: Edit form for deployment config
+            Given user has created deployment config workload
+              And user is at the Topology page
+             When user right clicks on the node to open context menu
+              And user selects option "Edit DeploymentConfig" from context menu
+             Then user will see option to switch between YAML view and Form view
+              And user will see "Timeout" section assciated with Recreate Strategy type under Deployment Strategy selected by default
+              And user will see "Timeout", "Maximum number of unavailable Pods" and "Maximum number of surge Pods" sections for Strategy type RollingUpdate under Deployment Strategy
+              And user will see "Image name", "Command" and "Environment variables(runtime only)" sections for Strategy type Custom under Deployment Strategy
+              And user will see "Deploy image from an image stream tag" checkbox under Images with project, image stream and tag section associated with it
+              And user will see Image name section under Images with checkbox "Deploy image from an image stream tag" unchecked
+              And user will see "Show additional parameters and lifecycle hooks", "Show advanced image options" and "Show advanced container options" in case of image stream option
+              And user will see advanced option for "Pause rollouts" and "Scaling"
+
+
+        @regression @manual
+        Scenario: Additional parameters and lifecycle hooks for deployment config
+            Given user is on Edit DeploymentConfig page
+             When user clicks on "Show additional parameters and lifecycle hooks" option
+              And user clicks on "Add Pre Cycle Hook" under Pre Cycle Hook section
+              And user clicks on "Add Mid Cycle Hook" under Mid Cycle Hook section
+              And user clicks on "Add Post Cycle Hook" under Post Cycle Hook section
+             Then user will see "Lifecycle action", "Container name", "Command", "Environment variables(runtime only)" "Volumes" and "Failure policy" sections in each of the form
+              And user will see Tick and cross buttons associated with the forms
+
+
+        @regression @manual
+        Scenario Outline: Advanced container options
+            Given user is on Edit "<resource>" page
```

Comment:
I think "user created deployment workload "workload-d" should be the first given statement. because if it is testing independently then the tester will be confused
We can automate this scenario right or any concerns?

Reply:
Updated